pkg/provisioner/reconciler/claim-reconciler/reconiler.go

```diff
@@ -144,6 +143,9 @@ func (r *ObjectBucketClaimReconciler) handleProvisionClaim(key client.ObjectKey,
 		return err
 	}
 
+	// use storage class to determine if this is a new or an old bucket
+	newBkt := scForNewBkt(class)
```

Comment:
bit of a nit - for readability down the line, the func name would be better spelled out.  `newBkt` makes me think it's a bucket object.  maybe something like `isProvisionedBucket` or `isNewBucket`